lib/middleware/karma.js

```diff
@@ -178,6 +178,7 @@ var createKarmaMiddleware = function (
             var file = files.included[i]
             var filePath = file.path
             var fileExt = path.extname(filePath)
+            var esModule = fileExt === '.js' && file.esModule
```

Comment:
`.endsWith('js')` might be better - there's [talk](https://github.com/nodejs/node-eps/blob/5dae5a537c2d56fbaf23aaf2ae9da15e74474021/002-es6-modules.md#51-determining-if-source-is-an-es-module) of changing the extension to 'mjs'